.github/workflows/global-ci-bundle.yml

```
@@ -556,7 +562,8 @@ jobs:
   koncur-kantra-tests:
     needs: [check-images, extract-koncur-ref]
     runs-on: ${{ matrix.runner }}
-    if: ${{ inputs.run_koncur_kantra_tests }}
+#   See https://github.com/konveyor/ci/issues/224
+    if: ${{ inputs.run_koncur_kantra_tests && (matrix.os != 'windows' || inputs.run_windows_kantra_tests) }}
```

actionlint (1.7.12)

[error] context "matrix" is not allowed here. available contexts are "github", "inputs", "needs", "vars". see https://docs.github.com/en/actions/learn-github-actions/contexts#context-availability for more details

(expression)